pkg/file/file.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 var (
-	ChunkWithLengthSize = swarm.ChunkSize + 8
+	ChunkWithSpanSize = swarm.ChunkSize + 8
```

Comment:
there is `swarm.SpanSize` constant too and if you need this should be in `swarm` pkg too

Reply:
@zelig , samen comment, same time :D